config/redirects.txt

```diff
@@ -13,3 +13,6 @@
 ^\/getting-started -> /raspberrypi/nodejs/getting-started
 
 ^\/configuration\/resin-base-images(\/|$) -> /runtime/resin-base-images
+
+# Renamed pages
+^\/deployment\/wifi\/?(.*)? -> /deployment/network/$1
```

Comment:
Yeah, looks good. The problem I'd tried to solve was if there are more URLs like

`/deployment/wifi-something` that will be captured here as well now.